suite-native/module-accounts-import/src/components/SelectableAssetList.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useMemo } from 'react';
+
+import { A, pipe } from '@mobily/ts-belt';
+
+import { Box, Card, Text, VStack } from '@suite-native/atoms';
+import { Network, networks, NetworkSymbol } from '@suite-common/wallet-config';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { SelectableAssetItem } from './SelectableAssetItem';
+
+type SelectableAssetListProps = {
+    items: Network[];
+    itemsOrder: NetworkSymbol[];
+    title: string;
+    onSelectItem: (currencySymbol: NetworkSymbol) => void;
+};
+
+const listItemTitleStyle = prepareNativeStyle(utils => ({
+    marginBottom: utils.spacings.medium,
+}));
+
+export const SelectableAssetList = ({
+    items,
+    itemsOrder,
+    title,
+    onSelectItem,
+}: SelectableAssetListProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    const networkAssetItem = useMemo(
+        () =>
+            pipe(
+                items,
+                A.map(network => ({
+                    cryptoCurrencySymbol: network.symbol,
+                    cryptoCurrencyName: networks[network.symbol].name,
+                    iconName: network.symbol,
+                    order: itemsOrder.includes(network.symbol)
+                        ? itemsOrder.indexOf(network.symbol)
+                        : Number.MAX_SAFE_INTEGER,
+                })),
+                A.sort((a, b) => (a.order > b.order ? 1 : -1)),
+            ),
+        [items, itemsOrder],
```

Comment:
Both `items` and `itemsOrder` are basically constants and there is very small chance we need differently anywhere, so I would suggest to remove these as props and just import them. Also then you can place this before component.

Reply:
b4d80e61f1d3d2f38840c1417c8cb8450de8156f